pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -204,6 +204,8 @@ self: super: {
   # Too strict bounds on optparse-applicative
   weeder = lib.warnIf (lib.versionAtLeast super.weeder.version "2.8.0") "jailbreak on weeder may be obsolete" doJailbreak super.weeder;
 
+  fused-effects = doJailbreak (markUnbroken super.fused-effects);
```

Comment:
`markUnbroken` is a function of last ressort. Please drop fused-effects from the `broken.yaml` and regenerate hackage-packages.nix.

Reply:
Also, a comment as to why you jailbreak would be welcome. Ideally with a link to an upstream issue.

Reply:
Thank you, will do! ❤️